File: python/CLUCKFOUR.py
z = int(input())
count = 0
while (z > 0):
    a = int(input())
    while (a > 0):
        b = int(a % 10)
        if (b == 4):
            count = count + 1
        a = a/10
    print(count)
    count = 0
    z -= 1

File: java/CLUCKFOUR.java
import java.util.Scanner;

public class CLUCKFOUR {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int z = sc.nextInt();
        int count = 0;
        while (z>0){
            int a = sc.nextInt();
            while (a>0){
                int b = a % 10;
                if(b==4){
                    count++;
                }
                a=a/10;
            }
            System.out.println(count);
            count=0;
            z--;
        }
    }
}
